src/main/java/baritone/behavior/PathingBehavior.java

```diff
@@ -565,4 +577,49 @@ private static AbstractNodeCostSearch createPathfinder(BlockPos start, Goal goal
     public void onRenderPass(RenderEvent event) {
         PathRenderer.render(event, this);
     }
+
+    @Override
+    public boolean isCloseToHome() {
+        return isCloseToHome;
+    }
+
+    @Override
+    public void updateIsCloseToHome() {
+        try {
+            IBaritone bar = BaritoneAPI.getProvider().getPrimaryBaritone();
```

Comment:
`PathingBehavior` extends from `Behavior`, which has a `public final Baritone baritone`. Use that here.

Reply:
Changed in latest commit